components/ExerciseCard.js

```diff
@@ -71,21 +91,37 @@ class ExerciseCard extends React.Component {
       this.props.darkTheme || false
         ? exerciseViewStyles.exersiseViewTextDark
         : exerciseViewStyles.exersiseViewTextLight;
+    const iconColor = this.props.darkTheme || false ? "#ff453a" : "#ff3b30";
 
     return (
       <View style={this.props.style}>
         <WorkoutCard style={exerciseViewStyle}>
-          <TextInput
-            style={[
-              exerciseViewTextStyle,
-              { ...FontStyles.h1, ...FontStyles.bold }
-            ]}
-            placeholder="Exercise name"
-            onChangeText={this.changeName}
-            underlineColorAndroid="transparent"
+          <View
+            style={{
+              display: "flex",
+              flexDirection: "row",
+              justifyContent: "space-between"
+            }}
           >
-            {this.props.exercise.name}
-          </TextInput>
+            <TextInput
+              style={[
+                exerciseViewTextStyle,
+                { ...FontStyles.h1, ...FontStyles.bold }
+              ]}
+              placeholder="Exercise name"
+              onChangeText={this.changeName}
+              underlineColorAndroid="transparent"
+            >
+              {this.props.exercise.name}
+            </TextInput>
+            <Icon
+              name="minus-circle"
```

Comment:
Not loving the icon, it stands out too much. Could you replace it with **[close](https://material.io/tools/icons/?search=close&icon=close&style=baseline)** icon?

Reply:
@AlexeiGitH i agree. Fixed. It looks better now.